components/create_comment/create_comment.tsx

```diff
@@ -1098,21 +1099,29 @@ class CreateComment extends React.PureComponent<Props, State> {
                 />
             );
             if (channelTimezoneCount > 0) {
+                const atHereMsg = 'By using **@here** you are about to send notifications to up to **{totalMembers} people** in **{timezones, number} {timezones, plural, one {timezone} other {timezones}}**. Are you sure you want to do this?';
```

Comment:
This is definitely outside the scope of this ticket, but since this logic is becoming more complicated, perhaps we could move the logic for rendering this modal into a new component called something like `NotifyConfirmModal` that could be controlled with the `openModal` redux action so that `CreateComment` and `CreatePost` don't have to be responsible for it? I'm thinking we could add something like
```javascript
this.props.actions.openModal({
    dialogType: NotifyConfirmModal,
    dialogProps: {
        mentions,
        onConfirm: () => {/*whatever actually submits the post*/},
    },
});
```
to the `handleSubmit` method instead of having it store a bunch of state in the component to make the modal appear.

0/5 on doing that now though

Reply:
I must admin that I've thought about that cause I am not a fan of redundant code but somehow when it comes to front-end work laziness gets the better of me. I'll have a stab at that.

Reply:
Sounds good. If not, that's totally fine too. I'm thinking of making an improved modal system as one of Web Platform's OKRs next quarter, so we could always address that then.